themes/custom/droopler_theme/js/global.js

```diff
@@ -62,108 +62,6 @@
-            var extendedElement = this.getExtendedPaddingElement();
-            var extendedElementSizing = Math.ceil(extendedElement.outerHeight() + extendedElement.position().top);
-
-            var sizingCompare = extendedElementSizing - Math.ceil(this.getElementPadding());
-
-            return sizingCompare > 0 ? sizingCompare : 0;
-          },
-          getElementPadding: function () {
-            var $clonedElement = $element.clone();
-
-            $clonedElement
-              .prop('style', '')
-              .css({
-                'position': 'absolute',
-                'z-index': -9999,
-                'top': -9999,
-                'left': -9999,
-              })
-              .appendTo($element.parent());
-
-            var clonedElementPadding = $clonedElement.css('padding-top');
-            $clonedElement.remove();
-
-            return parseFloat(clonedElementPadding);
-          },
-          isExtendedPaddingApplicable: function () {
-            return $element.offset().top < this.getExtendedPaddingElementOffset();
-          },
-          getCombinedPaddingCssValue: function () {
-            return this.getElementPadding() + this.getExtendedPaddingElementHeight() + 'px';
-          },
-          run: function () {
-            var header = this.getExtendedPaddingElement();
-
-            if (header.length  && $element.length) {
-              if (this.isExtendedPaddingApplicable()) {
-                $element.css('padding-top', this.getCombinedPaddingCssValue());
-                return this;
-              }
-              $element.removeAttr('style');
-            }
-
-            return this;
-          }
-        }
-
-        return paragraphPaddingApp.run();
-      };
-
-      var eventName = 'calculateDynamicHeaderOffset';
-
-      $(window)
-        .once(eventName)
-        .bind(eventName, Drupal.debounce(function () {
-          var firstParagraph = $('.paragraph-sections section:first > .paragraph:first');
-          var elementToObserve = firstParagraph;
-
-          if (firstParagraph.find('.content-inside-wrapper').length) {
-            elementToObserve = firstParagraph.find('.content-inside-wrapper');
-          }
-          else if (firstParagraph.find('.expandable-content .list-item-wrapper').length) {
-            if (!firstParagraph.hasClass('d-p-group-of-text-blocks') && !firstParagraph.hasClass('d-p-carousel')) {
-              firstParagraph.find('.expandable-content .list-item-wrapper section').each(function () {
-                paragraphPaddingObserver($(this));
-              });
-
-              return;
-            }
-          }
-          else if (firstParagraph.find('> .content-wrapper').length) {
-            elementToObserve = firstParagraph.find('> .content-wrapper');
-          }
-
-          paragraphPaddingObserver(elementToObserve);
-        }, 300));
-
-      $(window).on('resize',function () {
-        $(this).trigger(eventName);
-      }).trigger(eventName);
-    }
-  }
-
```

Comment:
uhm, ok, wywalam tutaj ten cały JS, bo on jest tylko od tego, żeby ustalić padding na pierwszym paragrafem - przeniosłem to do SCSS..